releasenotes/notes/add-kafka-distributed-tracing-150a31a858c2f7c5.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
```

Comment:
We're making some more hardware here - before we release this as a "feature", lets talk with Tracers, DSM, and APM eng/product.